sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,86 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.GrantPermissionRule
+import co.nimblehq.sample.compose.domain.model.Model
+import co.nimblehq.sample.compose.domain.usecase.UseCase
+import co.nimblehq.sample.compose.test.CoroutineTestRule
+import co.nimblehq.sample.compose.ui.AppDestination
+import co.nimblehq.sample.compose.ui.screens.MainActivity
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import org.junit.*
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+
+@ExperimentalTestApi
+@RunWith(AndroidJUnit4::class)
+class HomeScreenTest {
+
+    private val coroutinesRule = CoroutineTestRule()
```

Comment:
Was this intentional? Seems like this is not aligned with `HomeViewModelTest.kt` 💭 

```suggestion
    @get:Rule
    val coroutinesRule = CoroutineTestRule()
```

Reply:
@Tuubz Actually, we don't actually need to use a CoroutineTestRule as a `Rule` with `@get:Rule`. It will cause our UI tests failed with weird crashes as I don't get the reason this time.

![image](https://user-images.githubusercontent.com/16315358/218361459-4c468345-6c60-4595-b83f-d31b20e933d9.png)

- https://www.google.com/search?q=androidx.compose.ui.test.junit4.android.ComposeNotIdleException%3A+Idling+resource+timed+out%3A+possibly+due+to+compose+being+busy.&rlz=1C5CHFA_enVN966VN966&oq=androidx.compose.ui.test.junit4.android.ComposeNotIdleException%3A+Idling+resource+timed+out%3A+possibly+due+to+compose+being+busy.&aqs=chrome..69i57j69i59.1002j0j1&sourceid=chrome&ie=UTF-8
- https://issuetracker.google.com/issues/223815266
- https://github.com/RivuChk/Jetpack-Compose-MVI-Demo/issues/1
- https://issuetracker.google.com/issues/160259537

In anyways, we don't need to have `CoroutineTestRule`, I created minimized it to be only a `TestDispatchersProvider` object class 👌

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/d0a4ce710d805742b5160d44b28f657ed6b8f854